lib/workers/types.ts

```diff
@@ -131,7 +131,7 @@ export interface WorkerExtractConfig
   extends ExtractConfig,
     Partial<CustomExtractConfig> {
   manager: string;
-  fileList: string[];
+  fileList?: string[];
```

Comment:
But do we get this far if `fileList` is empty?

Reply:
In  `extractAllDependencies()` it doesn't matter if `fileList` is defined or not, as we create a `fileList` and filter it inside the function itself.

I looked at other places where `fileList` is used and at all places it was created using `getFileList()` or fs methods and not from config.

Reply:
This field was added in [this](https://github.com/RahulGautamSingh/renovate/commit/899b351643d65915391208713d61d3c8dd418654#diff-e04b1aa9f6115e15ba3c58cec0b8ce38870da32335f2f11d10181c43a27a06fe) PR. And it seems to me you added the addition so that we can use this property in managerConfig.